src/main/java/pl/hk/zadanie_31/config/ClientConfig.java

```diff
@@ -0,0 +1,21 @@
+package pl.hk.zadanie_31.config;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.web.client.RestTemplateBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.client.RestTemplate;
+
+@Configuration
+public class ClientConfig {
+
+    @Bean
+    @Autowired
+    public RestTemplate weatherRestTemplate(@Value("${https://api.openweathermap.org/data/2.5/weather}") String weatherHostUrl){
```

Comment:
To @Value oznacza, że chcielibyśmy pobrać wartość pola z pliku application.properties. Czyli jak dodamy w pliku application.properties linijke:

mojKlucz=test123

to mozemy użyć @Value("${mojKlucz}") String wartosc - i podstawi nam w to miejsce wartość test123. Dlatego w tym przypadku twój url do strony powinien być w pliku application.propertis, a tutaj tylko nazwa klucza